src/test/java/org/sagebionetworks/bridge/spring/controllers/AuthenticationControllerTest.java

```diff
@@ -1415,14 +1417,14 @@ private void mockJson(String json) throws Exception {
     
     private void verifyMetrics() {
         verify(controller, atLeastOnce()).getMetrics();
-        
         verify(metrics, atLeastOnce()).setSessionId(TEST_INTERNAL_SESSION_ID);
         verify(metrics, atLeastOnce()).setUserId(TEST_ACCOUNT_ID);
         verify(metrics, atLeastOnce()).setAppId(TEST_APP_ID);
     }
     
     private void verifyCommonLoggingForSignIns() throws Exception {
-        verifyMetrics();
+        // verifyMetrics();
```

Comment:
Is verifyMetrics() still being used? If not, can we delete it?

Reply:
`verifyMetric()` is still being used in `signOut` tests. I will delete this comment line but leave the `verifyMetric` as it is.